src/index.html

```diff
@@ -11,7 +11,169 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <div class="title1">
+        Personal  information
+      </div>
+      <div class="person">
+        <div class="person2">
+          <label>
+            Surname:
+            <input 
+            type="text" 
+            name="surname"
+            autocomplete="off">
```

Comment:
```suggestion
            <input 
              type="text" 
              name="surname"
              autocomplete="off"
            >
```
відформатуй всюди згідно чеклиста https://github.com/mate-academy/layout_html-form/blob/master/checklist.md